app/src/util.js

```diff
@@ -243,7 +243,26 @@
                     // TODO report error
                 }, this));
             }
+        },
+
+        // Display a bootstrap-style alert message to the user.
+        // Type should be success, info, warning, or danger.
+        // Timeout is how long the alert should be display.  Defaults to 5 seconds.
+        bootstrapAlert: function (type, message, timeout) {
+            timeout = typeof timeout !== 'undefined' ? timeout : 5;
+            timeout *= 1000; // convert to milliseconds
+
+            $('#alert_placeholder').html('<div id="alert" class="alert alert-' + type + ' alert-dismissable fade in"><button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button><span>' + message + '</span></div>');
```

Comment:
To avoid XSS issues, you should call `$.escapeHTML` on the `message` portion of this since it might be the case that the message contains user-created values like resource names or descriptions.

More generally, it's best to just avoid inlining HTML like this and instead defer to jade or another templating engine that will take care of escaping.